index.js

```diff
@@ -186,14 +206,11 @@ let specFiles;
 argv.fileIdentifiers = castArray(argv.fileIdentifiers);
 
 try {
-  specFiles = fs.readdirSync(cypressConfig.integrationFolder)
-    .filter(fileName => {
+  specFiles = glob.sync(cypressConfig.integrationFolder + '/**/*' + argv.fileFormat).filter(fileName => {
```

Comment:
This is kind of weird. So lets say we have two values:
- `integrationFolder = "cypress/config"`
-  `fileFormat = ".spec.js"`

Our glob pattern might look like this: `cypress/integration/**/*.spec.js`

We can probably use `testFiles` here, something like this:
```suggestion
  specFiles = glob.sync(cypressConfig.integrationFolder + '/' + argv.testFiles).filter(fileName => {
```

This would make the Glob pattern: `"cypress/integration/**/*.*"`.

That should work just fine too, and we can drop the `argv.fileFormat` CLI argument.

Reply:
Yep that should work 👍